frontend/src/features/apartment/ApartmentCreatePage.tsx

```diff
@@ -179,7 +165,7 @@ const ApartmentCreatePage = () => {
         }
     };
 
-    const [formData, setFormData] = useImmer<IApartmentWritable>(initialFormData);
+    const [formData, setFormData] = useImmer<IApartmentWritableForm>(initialFormData);
```

Comment:
Hmm.. I think there's no need for immer now that we're using react-hook-form. It state-variable itself seems to be used here only to load the initial value, so using `initialFormData` as is would probably be enough.
And the setter-function is only called upon a successful save operation.  When the form is submitted successfully it updates the values in the API automagically, so we have a new `initialFormData` anyway. I think there's no need to have a state-variable take care of the same thing.

Reply:
Good catch! I had plans to remove it, but looks like got distracted halfway through the process 😅